public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json

```
@@ -1979,7 +1979,7 @@
               "subscription": ""
             }
           ],
-          "title": "Max Available Bytes Recieved and Trend  Line",
+          "title": "Max Available Bytes Received and Trend  Line",
```

Thanks! I wonder if that extra space between `Trend` and `Line` was intentional..
```suggestion
          "title": "Max Available Bytes Received and Trend Line",
```

---

Thanks.
I wondered as well; probably not. I'll fix it.